sites/fast-website/src/public/index.html

```diff
@@ -110,7 +110,7 @@ <h2 class="section-heading">Join the community</h2>
                 <li><p>&copy; 2020 Microsoft</p></li>
               </ul>
             </site-navigation>
-            <fast-anchor class="logo" appearance="lightweight" href="/">
+            <fast-anchor class="logo" appearance="lightweight" href="/" aria-label="Fast logo">
```

Comment:
Does this direct users to the logo or does it direct them to the homepage? The label here should be descriptive of the link and not the visuals.

Reply:
Good point. Will change to "homepage".